### ISSUE TITLE
Server API: add capability to set/get guide parameters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#380]
```
User requests ability to get and set guiding parameters via server api

https://groups.google.com/forum/?fromgroups=#!topic/open-phd-guiding/WtaUAhpvzec

```

Original issue reported on code.google.com by `andy.gal...@gmail.com` on 21 Dec 2014 at 5:53

**Comment by agalasso** [comment]
another user requests ability to enable/disable guide output.

**Comment by agalasso** [comment]
fixed by d80608d2a4c6fa4da423486628c35d49808e33f8
